modules/microvm/virtualization/microvm/logvm.nix

```diff
@@ -0,0 +1,123 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}: let
+  configHost = config;
+  vmName = "log-vm";
+  macAddress = "02:00:00:01:01:02";
+  logvmBaseConfiguration = {
+    imports = [
+      (import ./common/vm-networking.nix {inherit vmName macAddress;})
+      ({lib, ...}: {
+        ghaf = {
+          users.accounts.enable = lib.mkDefault configHost.ghaf.users.accounts.enable;
+          development = {
+            # NOTE: SSH port also becomes accessible on the network interface
+            #       that has been passed through to NetVM
+            ssh.daemon.enable = lib.mkDefault configHost.ghaf.development.ssh.daemon.enable;
+            debug.tools.enable = lib.mkDefault configHost.ghaf.development.debug.tools.enable;
+            nix-setup.enable = lib.mkDefault configHost.ghaf.development.nix-setup.enable;
+          };
+          systemd = {
+            enable = true;
+            withName = "logvm-systemd";
+            withPolkit = true;
+            withDebug = configHost.ghaf.profiles.debug.enable;
+          };
+        };
+
+        system.stateVersion = lib.trivial.release;
+
+        nixpkgs.buildPlatform.system = configHost.nixpkgs.buildPlatform.system;
+        nixpkgs.hostPlatform.system = configHost.nixpkgs.hostPlatform.system;
+
+        environment.systemPackages = [pkgs.grafana-loki];
+
+        systemd.network = {
+          enable = true;
+          networks."10-ethint0" = {
+            matchConfig.MACAddress = macAddress;
+            addresses = [
+              {
+                addressConfig.Address = "192.168.100.66/24";
+              }
+              {
+                # IP-address for debugging subnet
+                addressConfig.Address = "192.168.101.66/24";
+              }
+            ];
+            linkConfig.ActivationPolicy = "always-up";
+          };
+        };
+
+        networking.firewall.allowedTCPPorts = [3100];
+
+        environment.etc."loki.yaml".source = ./loki-local-config.yaml;
+        systemd.services.loki = {
```

Comment:
Btw. I had to some fixes in my configuration after the last loki upgrade in nixpkgs unstable: https://github.com/Mic92/dotfiles/blob/main/nixos/eva/modules/loki.nix

Reply:
Thanks @Mic92 for pointing to correct settings.